docs/onclusterlayering-quickstart.md

```diff
@@ -24,39 +24,29 @@ spec:
   # image to be deployed to.
   machineConfigPool:
     name: layered
-  buildInputs:
-    containerFile:
-    # Here is where you can set the Containerfile for your MachineConfigPool.
-    # You'll note that you need to provide an architecture. This is because this
-    # will eventually support multiarch clusters. For now, only noArch is
-    # supported.
-    - containerfileArch: noarch
-      content: |-
-        <containerfile contents>
-    # Here is where you can select an image builder type. For now, we only
-    # support the "PodImageBuilder" type that we maintain ourselves. Future
-    # integrations can / will include other build system integrations.
-    imageBuilder:
-      imageBuilderType: PodImageBuilder
-    # The Machine OS Builder needs to know what pull secret it can use to pull
-    # the base OS image.
-    baseImagePullSecret:
-      name: <secret-name>
-    # Here is where you specify the name of the push secret you use to push
-    # your newly-built image to.
-    renderedImagePushSecret:
-      name: <secret-name>
-    # Here is where you specify the image registry to push your newly-built
-    # images to.
-    renderedImagePushspec: <final image pullspec>
-  buildOutputs:
-    # Here is where you specify what image will be used on all of your nodes to
-    # pull the newly-built image.
-    currentImagePullSecret:
-      name: <secret-name>
+  containerFile:
+  # Here is where you can set the Containerfile for your MachineConfigPool.
+  - content: |-
+      <containerfile contents>
+  # Here is where you can select an image builder type. For now, we only
+  # support the "Job" type that we maintain ourselves. Future integrations
+  # can / will include other build system integrations.
+  imageBuilder:
+    imageBuilderType: Job
+  # The Machine OS Builder needs to know what pull secret it can use to pull
+  # the base OS image.
+  baseImagePullSecret:
```

Comment:
this is optional now, if the user doesn't provide this we automatically copy the global-pull-secret for use. I think this is worth mentioning here and updating the rest of the doc with this information